interface/client/templates/popupWindows/onboardingScreen.js

```diff
@@ -81,7 +90,7 @@ Template['popupWindows_onboardingScreen'].helpers({
 
     @method syncStatus
     */
-    'syncStatus': function () {
+    syncStatus() {
```

Comment:
Thats ES6 right here, if we would remove the ecmascript package, this would break.